Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeHost.mm

```diff
@@ -295,8 +295,13 @@
 #if ENABLE(MODEL_ELEMENT)
     case PlatformCALayer::LayerTypeModelLayer:
 #endif
-    case PlatformCALayer::LayerTypeContentsProvidedLayer:
-        return RemoteLayerTreeNode::createWithPlainLayer(properties.layerID);
+    case PlatformCALayer::LayerTypeContentsProvidedLayer: {
+        auto layer = RemoteLayerTreeNode::createWithPlainLayer(properties.layerID);
+        // So that the scrolling thread's performance logging code can find all the tiles, mark this as being a tile.
+        if (properties.type == PlatformCALayer::LayerTypeTiledBackingTileLayer)
+            [layer->layer() setValue:@YES forKey:@"isTile"];
```

Comment:
I kind of wonder if we should add setValueForKey() on PlatformCALayer and move this logic out of here (and out of `PlatformCALayerCocoa::commonInit()`)

Reply:
But this is fine.

Reply:
Ah yeah that's a better solution. I'll try that.

Reply:
Kind of annoying though because it means maintaining a map of values and keys in PlatformCALayerRemote and serializing them and etc. etc. etc.

Probably better to just do your original patch.

Reply:
Yeah ended up being trickier than expected. This is probably fine.